MyXamarinApp.API/Models/EmployeeModel.cs

```diff
@@ -7,16 +8,22 @@ public class EmployeeModel
         /// <summary>
         /// Employee ID of the Employee.
         /// </summary>
-        public int EmpId { get; set; }
+        public Guid EmployeeGuid { get; set; }
 
         /// <summary>
         /// Full name of the Employee.
         /// </summary>
+        [Required]
         public string Name { get; set; }
 
         /// <summary>
         /// Role of the Employee.
         /// </summary>
+        [Required]
         public string Role { get; set; }
+
+        public DateTime JoiningDate { get; set; }
+        public bool IsActive { get; set; }
+        public string Salary { get; set; }
```

Comment:
Why is salary marked as string ?

Reply:
To show the amount with comma separated values.